daemon/daemon_unix.go

```diff
@@ -63,6 +72,10 @@ const (
 	// constant for cgroup drivers
 	cgroupFsDriver      = "cgroupfs"
 	cgroupSystemdDriver = "systemd"
+
+	// DefaultRuntimeName is the default runtime to be used by
+	// containerd if none is specified
+	DefaultRuntimeName = "docker-runc"
```

Comment:
How differs from DefaultRuntimeBinary?

Reply:
This is the name used for the --runtime option map